CONTRIBUTING.md

```diff
@@ -0,0 +1,620 @@
+
+- Referring to keyboard keys and keyboard shortcuts, see: [Microsoft Style Guide: Keys and keyboard shortcuts](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts).
+
+- Referring to touch interactions, such as "tap" and "swipe"; see: [Microsoft Style Guide: Describing alternative input methods - Pen-computing and touch procedures](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-alternative-input-methods#pen-computing-and-touch-procedures).
+
+## Adding admonitions or notes
+
+Avoid adding admonitions (also known as: *notices*, *notes*, *warnings*, or *callouts*) where possible. Over-use of admonitions may lead to what is sometimes referred to as "Admonition Fatigue". For an article on reason for limiting the use of admonitions, see: [Harvard Business Review: Consumer Warning Labels Aren’t Working](https://hbr.org/2016/11/consumer-warning-labels-arent-working).
+
+Admontions typically start with: *Tip*, *Note*, *Important*, *Caution*, or *Warning*.
+
+For example:
+
+This is a note; used for additional, indirectly related information.
+
+This is a note that is important for users to read.
+
+Tiny documentation uses four levels of admonition:
+
+1.  NOTE: Use for additional, indirectly related information. Do not use where it is possible to reword or rewrite the content to incorporate the information.
+
+2.  IMPORTANT: Use when ignoring the notice *may* or *will* lead to unexpected behavior.
+
+3.  CAUTION: Use when ignoring the notice *may* lead to one or more of the following:
+
+    - A significant increase in the risk of a security breach.
+
+    - A security vulnerability.
+
+    - Information loss.
+
+    - System failure.
+
+    - Worse outcomes than those listed here.
+
+4.  WARNING: Use when ignoring the notice *will* lead to one or more of the following:
+
+    - A significant increase in the risk of a security breach.
+
+    - A security vulnerability.
+
+    - Information loss.
+
+    - System failure.
+
+    - Worse outcomes than those listed here.
+
+## Useful general writing advice
+
+The following pages provide advice on writing, including technical
+writing.
+
+- For a brief overview of how users read (or don’t read) content, see: [Content Design London: Readability Guidelines - How people read](https://readabilityguidelines.co.uk/content-design/how-people-read/).
+
+- For points on how to structure content, see: [Content Design London: Readability Guidelines - Page design](https://readabilityguidelines.co.uk/content-design/page-design/).
+
+- For general information on writing, covering a variety of subjects, see: [plainlanguage.gov: Mary Dash’s Writing Tips](https://plainlanguage.gov/resources/articles/dash-writing-tips/).
+
+# Glossary
+
+Accessible Rich Internet Applications (ARIA): Acronym for *Accessible Rich Internet Applications*. Spell out and describe on first mention within a section.
```

Comment:
```suggestion
Accessible Rich Internet Applications (ARIA)
: Acronym for *Accessible Rich Internet Applications*. Spell out and describe on first mention within a section.
```
This section is meant to be a description list, so this should be on a new line.